### ISSUE TITLE
Use NLP to find Company Moat

### ISSUE THREAD

**Issue by DavidDr90** [DavidDr90/ValueInvestingTool#3]
try to find the moats for a given company symbol
we can use the SEC website or other businesses websites

**Comment by Fananos** [comment]
this is way more complicated, needs deep research abilities, don't even know where you would start with this one. Identifying moats is extremely difficult even for a person, and I can't think of a structured approach to it right now

**Comment by DavidDr90** [comment]
you are right, this is why this is a enhancement and we will need to consider if this is possible...
